src/components/ContinuationIn/AuthorizationInformation.vue

```diff
@@ -285,14 +285,28 @@ export default class AuthorizationInformation extends Mixins(DateMixin, Document
     if (!documentKey || !documentName) return // safety check
 
     this.isDownloading = true
-    await this.downloadDocument(documentKey, documentName).catch(error => {
+    const documentClass = 'CORP'
```

Comment:
Please create an enum for document classes.

Reply:
What do you think about an enum for the document type as well? (I'd be OK with both enums in the same file, fwiw.)

Reply:
Thanks for the cleanup. Looks good.